clean_cost_report_1.py

```diff
@@ -0,0 +1,167 @@
+                                     'total', 'cah', 'subprovider', 'longterm']]
+for i in range(len(templines)):
+    covar_list96.append(['S300001', templines[i], 600, tempnames[i]])
+
+covar_list96.append(['S300001', 1100, 600, 'q_nursery'])
+covar_list96.append(['S300001', 2900, 600, 'day_labor'])
+
+#convcenient way to get variables when you want the product set of the set of roots and the set of ends
+tempclmns = [700, 1000, 1100]
+templines = [1200, 1400, 1700, 2500]
+tempclmnnames = ['intern_res_', 'employees_payroll_', 'workers_nonpaid_']
+for j in range(len(tempclmns)):
+    tempnames = [tempclmnnames[j]+x for x in ['total', 'subprovider', 'longterm', 'total_plus']]
+    for i in range(len(templines)):
+        covar_list96.append(['S300001', templines[i], tempclmns[j], tempnames[i]])
+
+tempclmns = list(range(1200,1600,100))
+templines = [1200, 1400]
+tempclmnnames = ['_mchs', '_medicare', '_medicaid', '_total']
+for j in range(len(tempclmns)):
+    tempnames = [x+tempclmnnames[j] for x in ['discharge_total', 'discharge_subprovider']]
+    for i in range(len(templines)):
+        covar_list96.append(['S300001', templines[i], tempclmns[j], tempnames[i]])
+
+covar_list96.append(['S300001', 1700, 1500, 'discharge_longterm_total'])
+
+covar_list96.append(['S300002', 100, 500, 'avg_hr_wage_total'])
+covar_list96.append(['S300002', 400, 500, 'avg_hr_wage_phys_a'])
+covar_list96.append(['S300002', 500, 500, 'avg_hr_wage_phys_b'])
+covar_list96.append(['S300002', 501, 500, 'avg_hr_wage_nonphys_b'])
+covar_list96.append(['S300002', 600, 500, 'avg_hr_wage_res'])
+
+covar_list96.append(['S300002', 100, 400, 'hrs_total'])
+covar_list96.append(['S300002', 400, 400, 'hrs_phys_a'])
+covar_list96.append(['S300002', 500, 400, 'hrs_phys_b'])
+covar_list96.append(['S300002', 501, 400, 'hrs_nonphys_b'])
+covar_list96.append(['S300002', 600, 400, 'hrs_res'])
+
+covar_list96.append(['G200000', 2500, 100, 'patient_rev_inpatient'])
+covar_list96.append(['G200000', 2500, 200, 'patient_rev_outpatient'])
+covar_list96.append(['G200000', 2500, 300, 'patient_rev_total'])
+covar_list96.append(['G200000', 4000, 200, 'operating_exp_total'])
+
+covar_list96.append(['C000001', 10100, 600, 'charge_inpatient'])
+covar_list96.append(['C000001', 10100, 700, 'charge_outpatent'])
+covar_list96.append(['S100000', 3200, 100, 'uncompensated_cost'])
+
+wksht_list96 = []
+line_list96 = []
+clmn_list96 = []
+cost_varname_list96 = []
+
+for item in covar_list96:
+    wksht_list96.append(item[0])
+    line_list96.append(item[1])
+    clmn_list96.append(item[2])
+    cost_varname_list96.append(item[3])
+
+
+cost_var_list96 = [wksht_list96, line_list96, clmn_list96, cost_varname_list96]
+
+
+#####extract the list of desired variables from the raw dataset, for all desired years
+
+costdata96 = pd.DataFrame()
+
+for y in range(6,12):
+    costbasedata = pd.read_csv('medicare_cost_report/HOSPFY20'+str(y).zfill(2)+ '/hosp_20'+str(y).zfill(2)+ '_RPT.CSV', header=None, \
```

Comment:
When you get kinda messy string concatenation like this it's often time to use a helper function. Consider instead:
```
def cost_base_data_file_path(year, directory):
    year_folder = 'HOSPFY20'+str(year).zfill(2)
    base_data_file = 'hosp_20'+str(year).zfill(2)+ '_RPT.CSV'
    return directory + '/' + year_folder + '/' base_data_file

costbasedata = pd.read_csv(cost_base_data_file_path(y, 'medicare_cost_report'), ...)
```

Ditto for the other two files.